docs/cheaha/slurm/submitting_jobs.md

```diff
@@ -141,6 +141,18 @@ For more details on using `sbatch` please see the [official documentation](https
     If you are using bash or shell arrays, it is crucial to note they use 0-based indexing. Plan your `--array` flag indices accordingly.
 <!-- markdownlint-enable MD046 -->
 
+#### Throttling in Slurm Array Jobs
+
+Throttling in Slurm array jobs refers to limiting the number of concurrent jobs that can run simultaneously. This approach prevents the overloading of computing resources and ensures fair distribution of resources among users. From a performance perspective, throttling helps optimize overall job performance by reducing resource contention across the Cheaha cluster. When too many jobs run at the same time, they may compete for CPU, memory, or I/O, which can negatively impact performance. Please [contact us](../../index.md#how-to-contact-us) if you would prefer to disable throttling for larger array jobs.
```

Comment:
Instead of saying "prefer to disable..." let's simply leave the door open for growth.

That way we aren't sending the message that we will do things based on preference, but based on need. After all, we are the experts.

"Please [contact us](...) if your research needs exceed our capacity."

Reply:
Incorporated the changes.